README.md

```diff
@@ -1,36 +1,62 @@
+  <a href="https://slack.electricitymaps.com">
+    <img src="https://slack.electricitymaps.com/badge.svg" alt="PRs welcome!" /></a>
+  <a href="https://slack.electricitymaps.com">
+    <img src="https://img.shields.io/twitter/follow/ElectricityMaps" alt="Twitter Follow"
+    ></a>
+</p>
 
 ![image](https://app.electricitymaps.com/images/electricitymap_social_image.png)
 
-## Contribute
+## Introduction
 
-Thank you for your interest. Check out our general [contribution guidelines](Add link to the contribution guidelines).
+This project aims to provide a free, open-source, and transparent visualisation of the carbon intensity of electricity consumption around the world.
 
-Here are some of the ways you can contribute:
+We get the raw production data from a variety of sources, including official government data, and then run [our flow-tracing algorithm](https://www.electricitymaps.com/blog/flow-tracing) to calculate the actual carbon intensity of a country's electricity consumption.
 
-- [Building a new parser](https://github.com/electricitymaps/electricitymaps-contrib/wiki/Building-a-new-parser)
-- [Fixing a broken parser](https://github.com/electricitymaps/electricitymaps-contrib/wiki/Fixing-a-broken-parser)
-- [Changes to the frontend](https://github.com/electricitymaps/electricitymaps-contrib/wiki/Changes-to-the-frontend)
-- [Find data sources](https://github.com/electricitymaps/electricitymaps-contrib/wiki/Find-data-sources)
-- [Verify data sources](https://github.com/electricitymaps/electricitymaps-contrib/wiki/Verify-data-sources)
-- [Translating the app](https://github.com/electricitymaps/electricitymaps-contrib/wiki/Translating-electricitymaps.com)
-- [Updating region capacities](https://github.com/electricitymaps/electricitymaps-contrib/wiki/Update-region-capacities)
+_Try it out at [app.electricitymaps.com](https://app.electricitymaps.com), or download the app on [Google Play](https://play.google.com/store/apps/details?id=com.tmrow.electricitymap&utm_source=github) or [App store](https://itunes.apple.com/us/app/electricity-map/id1224594248&utm_source=github)._
 
-Join us on [Slack](https://slack.electricitymaps.com) if you wish to discuss development or need help to get started.
+## Contributing
 
-We would love your feedback on how to improve the contribution experience, please feel free to fill out this [form](https://forms.gle/VRWvEFwhtnhpzPVX8)
+The Electricity Maps app is a community project and we welcome contributions from anyone!
 
-## Contributions workflow
+We are always looking for help to build parsers for new countries, fix broken parsers, improve the frontend app, improve accuracy of data sources, discuss new potential data sources, update region capacities, and much more.
 
-If you wonder what happens to your contributions, check out this workflow of contributions review and acceptance.
+Read our [contribution guidelines](/CONTRIBUTING.md) to get started.
```

Comment:
We haven't actually merged this one yet. 😅
I'll get the final changes done today.

Reply:
Oh, didn't realise - will wait with merging this!

Reply:
Contributing.md is merged now so this should work just fine!